commcare_connect/web/templatetags/sort_link.py

```diff
@@ -0,0 +1,45 @@
+from django import template
+from django.utils.html import format_html
+
+register = template.Library()
+
+
+@register.simple_tag(takes_context=True)
+def sort_link(context, field, display_text):
+    request = context["request"]
+    current_sort = request.GET.get("sort", "name")
+
+    # Determine the new sorting order and icon based on the current sorting state
+    if current_sort == field:
+        new_sort = f"-{field}"
+        icon = "bi-arrow-up-square-fill"
+    elif current_sort == f"-{field}":
+        new_sort = field
+        icon = "bi-arrow-down-square-fill"
+    else:
+        new_sort = field
+        icon = "bi-arrow-down-short"
+
+    # Construct the URL with the new sorting parameter
+    url = f"{request.path}?sort={new_sort}"
+    if request.GET.get("page"):
+        url += f"&page={request.GET.get('page')}"
```

Comment:
nit: if you change the sorting it should probably go back to page 1 (it is unintuitive to start on page 3 of a different sort)

Reply:
Good point [9dd2cd6120236f5672f3b9d678caf0434b5c9434](https://github.com/dimagi/commcare-connect/pull/320/commits/9dd2cd6120236f5672f3b9d678caf0434b5c9434)